### ISSUE TITLE
make src rpms

### ISSUE THREAD

**Issue by stdweird** [quattor/release#117]
`mvn package` calls `rpmbuild` without path, so this is pretty easy:
- make script called `rpmbuild` somewhere

``` bash
#!/bin/bash
# First arg of rpmbuild in mvn package is -bb
shift
/usr/bin/rpmbuild -ba "$@"
exit $?
```
- add script to path

``` bash
export PATH=$PWD:$PATH
chmod +x rpmbuild
```
- run `mvn package`

**Comment by stdweird** [comment]
@jrha this produces both src and regular rpms

**Comment by jrha** [comment]
They get produced anyway, but I don't keep them because they are kind of pointless.

**Comment by stdweird** [comment]
how do you produce them?

**Comment by jrha** [comment]
Hmm, nope I was mistaken, we get the spec files though.

**Comment by stdweird** [comment]
yes, but maven makes no SOURCES, populates the buildroot directly, so spec files are a bit useless

**Comment by stdweird** [comment]
@jrha this is what i meant

**Comment by jrha** [comment]
Cool, thanks.

**Comment by jrha** [comment]
@wdpypere submitted a patch upstream for the maven rpm plugin, mojohaus/rpm-maven-plugin#66, if that gets merged we can avoid using @stdweird's workaround.

**Comment by wdpypere** [comment]
the patch just got merged at https://github.com/mojohaus/rpm-maven-plugin/pull/66

**Comment by ned21** [comment]
The PR is tagged for release in 2.2.0 but last release was 2.1.5 in Feb 2016. 😢
